apps/ca.c

```diff
@@ -1403,7 +1404,7 @@ static int certify_cert(X509 **xret, const char *infile, int certformat,
     EVP_PKEY *pktmp = NULL;
     int ok = -1, i;
 
-    if ((req = load_cert_pass(infile, certformat, passin, infile)) == NULL)
+    if ((req = load_cert_pass(infile, certformat, passin, "template certificate")) == NULL)
```

Comment:
That should be "certificate sign request"...  that's a term that's been around since the 90's and that "everyone" should be used too by now.  "template certificate" is...  yeah I understand the intention, but it's just not correct, and potentially misleading.

Reply:
At first I was also mislead by the name of the variable: `req`
but at this point no CSR is loaded but a cert form which some (template) data is copied).

The normal use case you apparently had in mind is reflected elsewhere in `ca.c`:
```
    req = load_csr(infile, informat, "certificate request");
```
but here confusingly it actually is a certificate the name of which is passed in via "-ss_cert":
```
    {"ss_cert", OPT_SS_CERT, '<', "File contains a self signed cert to sign"},
```

Reply:
I'll rename the confusing variable 'req' to 'template_cert' in `certify_cert()`.

Reply:
Ahhh, ok.  Yup, I misread that line, the function name is actually clear.  So, never mind

Reply:
> I'll rename the confusing variable ...

Good idea!

Reply:
Done

Reply:
> certificate sign request

pedantic nic: it's "certificate signing request" in the common usage I am familiar with, or if you want the protocol element name, RFC 2986 uses just "CertificationRequest".

Reply:
That's right. In the CMP context it's called Certification Request, too.

Reply:
BTW, we once had a customer how wanted to order a "singing service" :laughing: